components/admin_console/group_settings/__snapshots__/group_settings.test.jsx.snap

```diff
@@ -21,7 +21,7 @@ exports[`components/admin_console/group_settings/GroupSettings should match snap
     >
       <InjectIntl(FormattedMarkdownMessage)
         defaultMessage="Groups are a way to organize users and apply actions to all users within that group.
-For more information on Groups, please see [documentation](!https://about.mattermost.com)."
+For more information on Groups, please see [documentation](!http://www.mattermost.com/ad-ldap-groups)."
```

Comment:
@wiersgallak FYI - In the past, we had the redirect link as `...mattermost.com/default-<topic>` so that it was more clear for marketing and us that this was a default page and not an actual page on our website for that particular topic.

E.g., we could have mattermost.com/compliance on our website to give an overview of compliance, but have mattermost.com/default-compliance for our docs.

Reply:
Thanks @jasonblais @mkraft redirect has been updated to :http://www.mattermost.com/default-ad-ldap-groups. Can you please update the code?  Thank you!